test/unit/modules/cache.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+var node = require('./../../node.js');
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var async = require('async');
+var sinon = require('sinon');
+var modulesLoader = require('../../common/initModule').modulesLoader;
+var Cache = require('../../../modules/cache.js');
+
+describe('cache module', function () {
```

Comment:
only one function `setJsonForKey` from the cache module is covered by unit tests. Plese cover rest of them. Would be nice if you also stick with the current style of writing unit tests, using describe for start testing every module function, to have the following structure as a result: 
```
describe('cache'...
   describe('setJsonForKey'...
   describe('getJsonForKey'...
```